lib/elixir/lib/gen_server.ex

```diff
@@ -418,6 +418,11 @@ defmodule GenServer do
 
   If this callback is not implemented, the default implementation by
   `use GenServer` will return `{:noreply, state}`.
+
+  The default implementation catches all messages. If you override it,
```

Comment:
The GenServer always remove the message. If you don't implement the callback, it means the GenServer will fail for unknown messages, which may actually be desired. So if you don't want to define a catch-all clause, that's ok.